conf/config-default.yaml

```diff
@@ -14,122 +14,121 @@
-  # which is used under Tomcat.
-  # Turn this option on if you want to be compatible with servlet when matching URI path.
-  normalize_uri_like_servlet: false
+  delete_uri_tail_slash: false        # Delete the '/' at the end of the URI
+  normalize_uri_like_servlet: false   # If true, use the same path normalization rules as the Java 
+                                      # servlet specification. See https://github.com/jakartaee/servlet/blob/master/spec/src/main/asciidoc/servlet-spec-body.adoc#352-uri-path-canonicalization, which is used in Tomcat.
+  
   router:
-    http: radixtree_host_uri      # radixtree_uri: match route by uri(base on radixtree)
-                                  # radixtree_host_uri: match route by host + uri(base on radixtree)
-                                  # radixtree_uri_with_parameter: like radixtree_uri but match uri with parameters,
-                                  #   see https://github.com/api7/lua-resty-radixtree/#parameters-in-path for
-                                  #   more details.
-    ssl: radixtree_sni          # radixtree_sni: match route by SNI(base on radixtree)
-  #stream_proxy:                  # TCP/UDP proxy
-  #  only: true                   # use stream proxy only, don't enable HTTP stuff
-  #  tcp:                         # TCP proxy port list
-  #    - addr: 9100
-  #      tls: true
-  #    - addr: "127.0.0.1:9101"
-  #  udp:                         # UDP proxy port list
-  #    - 9200
-  #    - "127.0.0.1:9201"
-  #dns_resolver:                  # If not set, read from `/etc/resolv.conf`
-  #  - 1.1.1.1
-  #  - 8.8.8.8
-  #dns_resolver_valid: 30         # if given, override the TTL of the valid records. The unit is second.
-  resolver_timeout: 5             # resolver timeout
-  enable_resolv_search_opt: true  # enable search option in resolv.conf
+    http: radixtree_host_uri    # radixtree_host_uri: match route by host and URI
+                                # radixtree_uri: match route by URI
+                                # radixtree_uri_with_parameter: similar to radixtree_uri but match URI with parameters. See https://github.com/api7/lua-resty-radixtree/#parameters-in-path for more details.
+    ssl: radixtree_sni          # radixtree_sni: match route by SNI
+
+  # stream_proxy:                 # TCP/UDP L4 proxy
+  #   only: true                  # Enable L4 proxy only without L7 proxy.
+  #   tcp:
+  #     - addr: 9100              # TCP proxy listening ports 
+  #       tls: true
+  #     - addr: "127.0.0.1:9101"
+  #   udp:                        # UDP proxy listening ports 
+  #     - 9200
+  #     - "127.0.0.1:9201"
+  
+  # dns_resolver:                 # If not set, read from `/etc/resolv.conf`
+  #   - 1.1.1.1
+  #   - 8.8.8.8
+  # dns_resolver_valid: 30        # Override the default TTL of the DNS records.
+  resolver_timeout: 5             # Time that the server will wait for a response from the DNS resolver 
```

Comment:
why do we need to split comments into multiple lines?

Reply:
Style preference. Researching other styles now. 

Other reviewers: Feel free to share your thoughts on this as well.